sycl/source/detail/device_binary_image.cpp

```diff
@@ -235,6 +250,420 @@ DynRTDeviceBinaryImage::~DynRTDeviceBinaryImage() {
   Bin = nullptr;
 }
 
+// Exclusive property merge logic. It assumes there are no cases where
+// properties have different values and throws otherwise.
+template <typename RangeGetterT>
+static std::vector<sycl_device_binary_property>
+NaiveMergeBinaryProperties(const std::vector<const RTDeviceBinaryImage *> &Imgs,
```

Comment:
Function names in this file should start with lower case to be consistent with LLVM coding standards.

Reply:
They should have all been renamed now. Please let me know if you see one I've missed.